### PR TITLE
Gh pages

### PR DESCRIPTION
**Pull request by rkoshak**
Some initial edits while finding my way around the documents. The primary focus has been on simplifying some of the wording and sentence structure to make them more user focused and less developer focused where reasonable. I also change the overview to talk about openHAB instead of ESH.

Signed-off-by: Rich Koshak <rlkoshak@gmail.com> (github: rkoshak)

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (kaikreuzer) [inline index.md]: This page has already completely changed on HEAD, so please don't overwrite it.

File: concepts/index.md [hunk 2]
```diff
@@ -4,34 +4,34 @@ layout: documentation
```
Comment (kaikreuzer) [inline concepts/index.md]: This page is copied over during the Maven build - it must not be manually edited (see https://github.com/openhab/openhab-docs#automatically-generated-parts).

What you can do is to create an openHAB version of it, put it in some other folder and update the menu to point to your version.

File: overview.md [hunk 3]
```diff
@@ -6,36 +6,44 @@ layout: documentation
...
 
 The center of this community is the [openHAB forum](https://community.openhab.org).
 
-If you have questions about the use of openHAB, before reporting an issue, or for comments and contributions about the documentation, please check the [openHAB forum](https://community.openhab.org) for solutions and help. More people will see your issue when it is posted in the forum and community members may be able to help you find known solutions/workarounds for your issue.
+Please post questions or comments to the [openHAB forum](https://community.openhab.org) first when encountering problems. The forum is populated with many helpful openHAB users and developers who can assist with almost any topic related to openHAB including installation, troubleshooting runtime problems, identifying bugs, explaining parts of the system, working with specific add-ons, working with rules and configurations, and potential solutions and workarounds for problems. There are also many excellent examples rules, configurations, and tutorials posted to the forum.
 
 Rules about posting an issue and [contributions guidelines](appendix/contributing.html) are in the _Appendix_.
 
+<<<<<<< HEAD
```
Comment (kaikreuzer) [inline overview.md]: this looks like an accidental change

File: concepts/index.md (hunk 2 above)
Comment (kaikreuzer) [inline concepts/index.md]: Note that this page was also updated by https://github.com/eclipse/smarthome/pull/1573 which was merged today - it does not speak about ESH anymore.

**Comment by kaikreuzer** [comment]
@rkoshak Did you see my comments?

**Comment by rkoshak** [comment]
Sorry, just got back home today. The network back in the islands in Nova
Scotia was barely enough to check my email, and sometimes not even then.
Now that I'm back home I can catch up with everything.

**Comment by kaikreuzer** [comment]
Ah, right, I completely forgot that you were on holiday - welcome back :-)

**Comment by rkoshak** [comment]
Given all the recent changes and the fact this is automatically copied over (something I missed when I started) I think this PR is overcome by events. What is the proper procedure for rejecting the PR and closing this out? Do I just close it?

**Comment by kaikreuzer** [comment]
Yes, it is indeed that easy :-)

**Pull request closed without merging**